apps/front-end/back-office-e2e/playwright.docker.config.ts

```diff
@@ -0,0 +1,12 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+import { baseConfig } from '../../../playwright.config.base';
+
+const config: PlaywrightTestConfig = {
+  ...baseConfig,
+  use: {
+    ...baseConfig.use,
```

Comment:
Cannot this be removed?

Reply:
No when using docker we must not be loading global config